src/charts/sparkline/sparkline-chart.directive.js

```diff
@@ -0,0 +1,327 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.charts.directive:pfSparklineChart
+ *
+ * @description
+ *   Directive for rendering a sparkline chart.
+ *   <br><br>
+ *   See http://c3js.org/reference.html for a full list of C3 chart options.
+ *
+ * @param {object} config configuration settings for the sparkline chart:<br/>
+ * <ul style='list-style-type: none'>
+ * <li>.chartId            - the ID iof the container that the chart should bind to
```

Comment:
Hi Walden,
How do you get the code snippet to appear in the Conversation here?  I goto the commit, click on the blue +, but I don't get code snippets to appear here
Thanks,
- Dave